src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/data/AdjudicationHearingMappingDto.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.data
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.Size
+import uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.AdjudicationHearingMapping
+import java.time.LocalDateTime
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Adjudication hearing mapping")
+data class AdjudicationHearingMappingDto(
+
+  @Schema(description = "NOMIS hearing id", required = true, example = "123456")
+  val nomisHearingId: Long,
+
+  @Schema(description = "DPS hearing id", required = true, example = "123456")
+  val dpsHearingId: String,
+
+  @Schema(description = "Label (a timestamp for migrated ids)")
+  @field:Size(max = 20)
+  val label: String? = null,
+
+  @Schema(
+    description = "Mapping type",
+    allowableValues = ["MIGRATED", "ADJUDICATION_CREATED"],
+    defaultValue = "ADJUDICATION_CREATED",
+  )
+  @field:Size(max = 20)
+  val mappingType: String? = null,
```

Comment:
Is that defaultValue above related the default here?

Reply:
Yes and no. There are 2 ways to create this mapping, via migration using the DTO above so this is nested. That will always be MIGRATED, or on it own (the endpoint hasn't been written yet) that will default to "ADJUDICATION_CREATED"